go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/lib/pq v1.10.7
 	github.com/miguelmota/go-solidity-sha3 v0.1.1
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/onsi/gomega v1.30.0
```

Comment:
By the way not sure whether we needed to introduce that dependency. Based on how it is used, it shouldn't be too much of an effort to implement a function that would periodically execute some custom logic and within its body, we could make assertions using `testify` lib and make certain assertions. WDYT?

Reply:
Done